README.md

```diff
@@ -1136,37 +1229,50 @@ async def run():
 
 
 if __name__ == "__main__":
-    import asyncio
-
     asyncio.run(run())
 ```
 
+_Full example: [examples/snippets/servers/lowlevel/basic.py](https://github.com/modelcontextprotocol/python-sdk/blob/main/examples/snippets/servers/lowlevel/basic.py)_
+<!-- /snippet-source -->
+
 Caution: The `uv run mcp run` and `uv run mcp dev` tool doesn't support low-level server.
 
 #### Structured Output Support
 
 The low-level server supports structured output for tools, allowing you to return both human-readable content and machine-readable structured data. Tools can define an `outputSchema` to validate their structured output:
 
+<!-- snippet-source examples/snippets/servers/lowlevel/structured_output.py -->
 ```python
-from types import Any
+"""Low-level server example showing structured output support.
 
+This example demonstrates how to use the low-level server API to return
+structured data from tools, with automatic validation against output schemas.
+
+Run from the repository root:
+    uv run examples/snippets/servers/lowlevel/structured_output.py
+"""
```

Comment:
Those module docstring seems to be adding a lot of unnecessary content to the README, given that you can run each of those individually.